src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -891,6 +903,7 @@ protected void setPowThreads(int powThreads) {
 
         //PearlDiver
         int POW_THREADS = 0;
+        String BROKER_HOSTNAME = "localhost";
```

Comment:
Should the default hostname localhost?

Reply:
Or maybe we can leave the default value as an empty `String`.

Reply:
I decide to use the `localhost` as default value.